libs/community/langchain_community/chat_models/azureml_endpoint.py

```diff
@@ -1,35 +1,44 @@
 import json
-from typing import Any, Dict, List, Optional, cast
+from typing import Any, Dict, List, Optional, cast, Iterator, AsyncIterator, Type, Mapping
 
-from langchain_core.callbacks.manager import CallbackManagerForLLMRun
+from langchain_core.callbacks import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
 from langchain_core.language_models.chat_models import BaseChatModel
 from langchain_core.messages import (
     AIMessage,
+    AIMessageChunk,
     BaseMessage,
+    BaseMessageChunk,
     ChatMessage,
+    ChatMessageChunk,
     HumanMessage,
+    HumanMessageChunk,
     SystemMessage,
+    SystemMessageChunk,
+    FunctionMessageChunk,
+    ToolMessageChunk,
 )
-from langchain_core.outputs import ChatGeneration, ChatResult
-
+from langchain_core.outputs import ChatGeneration, ChatResult, ChatGenerationChunk
 from langchain_community.llms.azureml_endpoint import (
     AzureMLBaseEndpoint,
     AzureMLEndpointApiType,
     ContentFormatterBase,
 )
 
 
-class LlamaContentFormatter(ContentFormatterBase):
+class OpenAIStyleContentFormatter(ContentFormatterBase):
```

Comment:
This is not correct. `LlamaContentFormatter` is kept only for backward compatibility. You should not rename it. It's deprecated.

Reply:
Updated